.eslintrc.js

```diff
@@ -2,6 +2,11 @@ const { overrides } = require('@netlify/eslint-config-node')
 
 module.exports = {
   extends: '@netlify/eslint-config-node',
+  rules: {
+    // This rule enforces using Buffers with `JSON.parse()`. However, TypeScript
+    // does not recognize yet that `JSON.parse()` accepts Buffers as argument.
```

Comment:
Due to the above, we pass `utf8` when in TypeScript, but not when in JavaScript.